puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman-discovery-image.groovy

```diff
@@ -0,0 +1,54 @@
+pipeline {
+    agent { label 'admin && sshkey' }
+
+    options {
+        timestamps()
+        timeout(time: 3, unit: 'HOURS')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage('Run FDI build') {
+            steps {
+                deleteDir()
+                script {
+                    job_parameters = [
+                        proxy_repository: env.proxy_repository,
+                        branch: env.branch,
+                    ]
+                    runCicoJob("foreman-discovery-image-build", job_parameters)
```

Comment:
so turns out, `runCicoJob` calls `deleteDir()` and thus we never see the artifacts :/